Frameworks/UIKit/UIImage.mm

```diff
@@ -69,7 +69,9 @@ void UIImageSetLayerContents(CALayer* layer, UIImage* image) {
     if ([layer contentsOrientation] != [image imageOrientation]) {
         [layer setContentsOrientation:[image imageOrientation]];
     }
-    CGRect stretch = [image _imageStretch];
```

Comment:
>stretch [](start = 11, length = 7)

init stretch to CGRectZero should avoid the issue as well. i have seen some code pattern like that in our code base.